pom.xml

```diff
@@ -63,9 +64,15 @@
         </dependency>
 
         <dependency>
-            <groupId>ch.sbb.polarion.thirdparty.bundles</groupId>
-            <artifactId>org.apache.poi</artifactId>
-            <version>${ch.sbb.polarion.thirdparty.bundles.org.apache.poi.version}</version>
+            <groupId>org.apache.poi</groupId>
```

Comment:
Wouldn't be this a problem to compile against one version of apache-poi, but run against another one?